lib/kitchen/directions/bake_math_in_paragraph.rb

```diff
@@ -2,7 +2,7 @@ module Kitchen
   module Directions
     module BakeMathInParagraph
       def self.v1(book:)
-        book.search('//p//math | //p//m').each do |math|
+        book.search('p m|math', '//p//math | //p//m').each do |math|
```

Comment:
rubocop / lefthook are maybe not running because this shouldn't have been pushable without the frozen string literal comment at the top.

Reply:
Rubocop is running, but it hasn't been forcing me to resolve all issues before pushing